datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/licenses.py

```diff
@@ -252,17 +247,21 @@ def update_copyrights(package_name, license_id, data, ctx):
     """
     Update package data with scraped copyright attributions.
     """
-    home_page = data['home_page']
-    repo_url = PACKAGE_REPO_OVERRIDES.get(package_name)
-    created_date = ''
-
-    if not repo_url:
-        repo_url = home_page
 
-    if repo_url:
-        cp, created_date = scrape_copyright_data(repo_url, ctx)
-        if cp:
-            data['copyright'][license_id] = cp
+    gh_repo_url = PACKAGE_REPO_OVERRIDES.get(package_name)
+    gh_repo_url = gh_repo_url if gh_repo_url else data['home_page']
+    if gh_repo_url is not None:
+        created_date = probe_github(gh_repo_url, ctx)
+
+    pkg_urls = [u for u in (d['url'] for d in data['urls']) if u.endswith('.whl') or u.endswith('.tar.gz')]
+    if not pkg_urls:
+        raise ValueError(
+            f"Found no urls to packages, here are the urls for this dependency: {[u['url'] for u in data['urls']]}"
+        )
+    url = pkg_urls[0]  # Since we only scan the contents of the package archive we don't care which package we download.
```

Comment:
Let's not use inline comments:

```suggestion

    # Since we only scan the contents of the package archive we don't care which package we download.
    url = pkg_urls[0]
```